src/main/java/bio/terra/testrunner/runner/TestRunner.java

```diff
@@ -513,6 +511,27 @@ protected void writeOutResults(String outputParentDirName) throws IOException {
     logger.info("Version script result written to file: {}", terraVersionFile.getName());
   }
 
+  // A mapper that unnest List<TestScriptResult> from TestScriptResult
+  private List<Map<String, Object>> unnestUserJourneyResults(
```

Comment:
This is much easier to follow, thanks! Can you move this to the `TestRunFullOutput` class and call it e.g. from the constructor instead of adding to `TestRunner`?

Reply:
Refactored `unnestUserJourneyResults` into `TestRunFullOutput`. Below is the output of the `TestRunFullOutput.testScriptResults` upon a test run.

```
  "testScriptResults" : [ {
    "userJourneyResults" : [ {
      "userJourneyDescription" : "ServiceStatus: 150",
      "threadName" : "pool-2-thread-1",
      "completed" : true,
      "elapsedTimeNS" : 150608732872,
      "exceptionWasThrown" : false,
      "exceptionStackTrace" : null,
      "exceptionMessage" : null
    }, {
      "userJourneyDescription" : "ServiceStatus: 150",
      "threadName" : "pool-2-thread-2",
      "completed" : true,
      "elapsedTimeNS" : 150608761870,
      "exceptionWasThrown" : false,
      "exceptionStackTrace" : null,
      "exceptionMessage" : null
    } ],
    "testScriptName" : "ServiceStatus",
    "testScriptDescription" : "ServiceStatus: 150"
  } ],

```